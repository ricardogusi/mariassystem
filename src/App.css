/* 
font-family: 'Ranchers', cursive; 
font-family: 'Raleway', sans-serif;
*/

/* 
Paleta de cores:
A #FB962A
B #FFCB94
C #E0A86C
D #297894
E #6CC1E0

*/

.container {
  width: 100%;
  height: 100vh;
  background: #297894;
  display: flex;
  align-items: center;
  justify-content: center;
  flex-direction: column;
}

h1 {
  margin: 0;
  font-size: 3rem;
  font-family: "Ranchers", cursive;
  color: #fb962a;
}




