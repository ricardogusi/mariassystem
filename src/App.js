import React from "react";
import "./App.css";

import CardData from "./Components/CardData/CardData";

function App() {
  return (
    <div className="container">
      <CardData/>
    </div>
  );
}

export default App;
