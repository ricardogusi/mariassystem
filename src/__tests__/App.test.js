import React from 'react';
import * as ReactDom from 'react-dom'
import renderer from 'react-test-renderer'
import App from '../App'



it('Renderiza corretamente', () => {
    const tree = renderer
      .create(<App /> )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('Verifica se a div possui a classe correta', ()=>{
    const root = document.createElement('div');
    ReactDom.render(<App />, root);
    expect(root.querySelector('div').className).toBe('container')

})