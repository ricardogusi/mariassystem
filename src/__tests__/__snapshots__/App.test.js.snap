// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renderiza corretamente 1`] = `
<div
  className="container"
>
  <h1>
    Just do it!
  </h1>
  <button
    className="adicionarButton"
    onClick={[Function]}
  >
    Adicionar Jogo
  </button>
  <div
    className="header"
  >
    <h3>
      Jogo
    </h3>
    <h3>
      Placar
    </h3>
    <h3>
      Mín da Temporada
    </h3>
    <h3>
      Máx da Temporada
    </h3>
    <h3>
      Quebra recorde Mín
    </h3>
    <h3>
      Quebra recorde Máx
    </h3>
  </div>
  <div
    className="card"
  >
    <div
      className="data"
    >
      <p>
        # 
        4
      </p>
      <p>
        24
      </p>
      <p>
        10
      </p>
      <p>
        24
      </p>
      <p>
        1
      </p>
      <p>
        1
      </p>
    </div>
    <div
      className="data"
    >
      <p>
        # 
        3
      </p>
      <p>
        10
      </p>
      <p>
        10
      </p>
      <p>
        24
      </p>
      <p>
        1
      </p>
      <p>
        1
      </p>
    </div>
    <div
      className="data"
    >
      <p>
        # 
        2
      </p>
      <p>
        24
      </p>
      <p>
        12
      </p>
      <p>
        24
      </p>
      <p>
        0
      </p>
      <p>
        1
      </p>
    </div>
    <div
      className="data"
    >
      <p>
        # 
        1
      </p>
      <p>
        12
      </p>
      <p>
        12
      </p>
      <p>
        12
      </p>
      <p>
        0
      </p>
      <p>
        0
      </p>
    </div>
  </div>
</div>
`;
